fil-proofs-tooling/Cargo.toml

```diff
@@ -31,14 +31,14 @@ storage-proofs = { path = "../storage-proofs"}
 filecoin-proofs = { path = "../filecoin-proofs"}
 tempfile = "3.0.8"
 cpu-time = "1.0.0"
-git2 = "0.10.1"
-heim = "0.0.9"
-futures-preview = "0.3.0-alpha.17"
+git2 = "0.13.6"
+heim = { version = "0.1.0-alpha.1", features = ["full", "runtime-tokio"] }
```

Comment:
There seems to be a new version of `heim`. Also it looks like `heim` has support for async-std. In Rust Filecoin stuff we use async-std, rather than tokio. Do you think it would be hard to use the async-std instead?

PS: I'm not fully done with the review yet, but I thought I give early feedback on things I found.